GrowthBook.Tests/ApiTests/LoadFeaturesTests.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace GrowthBook.Tests.ApiTests;
+
+public class LoadFeaturesTests
+{
+    private const string ApiKey = "<AN API KEY GOES HERE>";
```

Comment:
If we're including this file I'd recommend using IConfiguration for these things:
See, [this](https://dev.to/andreav/using-dotnet-iconfiguration-inside-xunit-projects-4o6b) for an example.

Reply:
File is removed now.